Documentation/Features.md

```diff
@@ -0,0 +1,34 @@
+# WheelMUD Features
```

Comment:
Fully tested? Basically none. Yes, I suspect some of those things are broken or disabled.